src/framework/ui/drawer/drawer.component.tsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import {
+  GestureResponderEvent,
+  ListRenderItemInfo,
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  styled,
+  StyledComponentProps,
+  StyleType,
+} from '@kitten/theme';
+import {
+  DrawerItem,
+  DrawerItemElement,
+  DrawerItemProps,
+} from './drawerItem.component';
+import {
+  List,
+  ListProps,
+} from '../list/list.component';
+import { Override } from '../support/typings';
+
+export type DrawerHeaderElement = React.ReactElement<any>;
+
+type DrawerListProps = Override<ListProps, {
+  data: DrawerItemProps[],
+  renderItem?: (info: ListRenderItemInfo<DrawerItemProps>) => DrawerItemElement;
+}>;
+
+interface ComponentProps extends DrawerListProps {
```

Comment:
Is it make sense to add a property like "footer"? Looks like SafeAreaView doesn't handle the bottom iOS and Android controls or cut-offs.